prov/efa/src/rdm/efa_rdm_cq.c

```diff
@@ -94,13 +94,18 @@ static ssize_t efa_rdm_cq_readfrom(struct fid_cq *cq_fid, void *buf, size_t coun
 
 	ofi_genlock_lock(srx_ctx->lock);
 
-	if (cq->shm_cq)
+	if (cq->shm_cq) {
 		fi_cq_read(cq->shm_cq, NULL, 0);
 
-	ret = ofi_cq_read_entries(&cq->util_cq, buf, count, src_addr);
+		/* 
+		 * This call aims to check whether there are completions written
```

Comment:
This comment is not very clear.  I think it would be better to explain why would we want to call `ofi_cq_read_entries` after calling `fi_cq_read` on shm_cq

Reply:
Updated comment